src/test/debuginfo/function-prologue-stepping-no-stack-check.rs

```diff
@@ -37,100 +37,79 @@
 // gdb-command:run
 
 // IMMEDIATE ARGS
-// gdb-command:print a
-// gdb-check:$1 = 1
-// gdb-command:print b
-// gdb-check:$2 = true
-// gdb-command:print c
-// gdb-check:$3 = 2.5
+// gdb-command:info args
```

Comment:
OK, makes sense. However, this is exactly what these test cases are for: That `print arg` will print the correct value when setting a function breakpoint. So using `info args` is a bit like cheating `:P`